BUILDING.md

```diff
@@ -1,8 +1,8 @@
 gRPC C++ - Building from source
 ===========================
 
-This document has detailed instructions on how to build gRPC C++ from source. Note that it only covers the build of gRPC itself and is mostly meant for gRPC C++ contributors and/or power users.
-Other should follow the user instructions. See the [How to use](https://github.com/grpc/grpc/tree/master/src/cpp#to-start-using-grpc-c) instructions for guidance on how to add gRPC as a dependency to a C++ application (there are several ways and system wide installation is often not the best choice).
+This document has detailed instructions on how to build gRPC C++ from the source. Note that it only covers the build of gRPC itself and is meant for gRPC C++ contributors and/or power users. 
```

Comment:
I don't think the word "the" is needed here.  "Building from source" is the way this is typically said.

Reply:
Done. Thanks for suggesting 😀